public/src/home/publicity-panel.html

```diff
@@ -57,11 +57,9 @@
       }
     </style>
     <div class="content">
-      <h2>Early Publisher Bonus</h2>
-      <p class="row">We're offering daily cash bonuses to early content creators. For the first 48 hours after posting, in addition to what
-        you earn from viewers we will give you
-        <span class="price">[[newUserBonus]]</span> for each viewer. There is
-        <span class="price">[[remaining]]</span> remaining today.
+      <h2>Early Publisher Bonuses</h2>
+      <p class="row">As a card creator, in addition to what you earn from viewers, we are currently paying you
+        <span class="price">[[newUserBonus]]</span> for each viewer.
         <a href="https://channels.cc/c/cecb7191-9956-4a88-a45d-34c99247cdbf" target="_blank">Learn more.</a>
```

Comment:
Minor personal preference: This sounds like it is like talking to existing publishers only. Previous wordings sounded like incentives to become a publisher. 
Don't care much either way though.

Reply:
Okay.  I've rewritten it.  See if you like the revised version any better.  (Or suggest another version.)

Reply:
It's fine. I am not sure what I really want. Good to go for now.